frontend/src/views/funding/FundingOverviewView.vue

```diff
@@ -25,21 +25,131 @@
           Funding Re-allocation
         </v-tab>
       </v-tabs>
-
       <v-card-text>
         <v-window v-model="tab">
           <v-window-item value="agreements">
-            <h2>Funding Details</h2>
-            <v-data-table :headers="headers" :items="fundingAgreements" item-key="guid" :items-per-page="10" density="compact">
-              <template #[`item.actions`]="{ item }">
-                <v-btn v-if="item.status === 'FA Signature Pending'" variant="text" @click="this.$router.push({ name: 'funding', params: { fundingGuid: item.guid } })">
-                  <v-icon size="large">mdi-signature-freehand</v-icon>
-                </v-btn>
-                <v-btn v-if="item.status === 'FA Submitted to Ministry'" variant="text" @click="this.$router.push({ name: 'funding', params: { fundingGuid: item.guid } })">
-                  <v-icon size="large">mdi-folder-open-outline</v-icon>
-                </v-btn>
-              </template>
-            </v-data-table>
+            <v-form ref="faForm" v-model="isFormComplete">
```

Comment:
I think the UI needs a little work at lower resolutions:

![image](https://github.com/bcgov/ecc-ofm/assets/80127167/588e6c51-abc2-49eb-8155-42ddc36fd322)

Reply:
Fixed. Yikes, i seem to always under estimate this.

Reply:
Much better but the date range is getting cut off quite early.

![image](https://github.com/bcgov/ecc-ofm/assets/80127167/7ec09280-cdb1-4372-821a-7f8414cde351)

Reply:
fixed but has to break quite to single col quite early. Its like there needs to be a sm **md** & lg, not just sm and lg.